tests/test_extensions/test_sklearn_extension/test_sklearn_extension.py

```diff
@@ -311,6 +333,23 @@ def test_serialize_model_with_subcomponent(self):
         self.assertEqual(new_model_params, model_params)
         new_model.fit(self.X, self.y)
 
+        xml = serialization._to_dict()
+        new_model2 = self.extension.flow_to_model(OpenMLFlow._from_dict(xml))
```

Comment:
Is this to primarily check the ```_from_dict``` function?
If so, given this unit test is really a long one, and since *xml* only depends on a ```model_to_flow``` call, does it make sense to have a smaller unit test checking this?

Reply:
> Is this to primarily check the _from_dict function?

Yes

> If so, given this unit test is really a long one, and since xml only depends on a model_to_flow call, does it make sense to have a smaller unit test checking this?

Yes and no. I'll think about whether this can easily be done without adding too much boilerplate code.